backend/btrixcloud/background_jobs.py

```diff
@@ -213,6 +217,14 @@ async def job_finished(
         if success:
             if job_type == BgJobType.CREATE_REPLICA:
                 await self.handle_replica_job_finished(cast(CreateReplicaJob, job))
+        else:
+            print(
+                f"Background job {job.id} failed, sending email to superuser",
+                flush=True,
+            )
+            superuser = await self.user_manager.get_superuser()
+            org = await self.org_ops.get_org_by_id(job.oid)
+            self.email.send_background_job_failed(job, org, finished, superuser.email)
```

Comment:
Just in case, should wrap this in `run_in_executor()` so it doesn't block the async event loop handling for the operator.

Reply:
Now added!